static/src/javascripts/projects/common/modules/article/rich-links.js

```diff
@@ -81,8 +81,11 @@ define([
             !isSensitive &&
             !isDuplicate
         ) {
+            if (typeof richLinkTagTmpl === 'string') {
+                richLinkTagTmpl = template(richLinkTagTmpl);
+            }
```

Comment:
@OliverJAsh oh sorry, I tend to see vars as empty vessels and just put anything inside, but you're right it's not the clearest way of writing code. You're right, unless templates were somehow generated at runtime, there's no reason not to pre-compile that during r.js optimisation—and even then, this kind of work is worth offloading to a web worker. I'll give the lodash loader a try, otherwise a slightly modified version of the text plugin should work too.